run_benchmarks.sh

```diff
@@ -9,6 +9,10 @@
 # at your option. This file may not be copied, modified, or distributed except according to those
 # terms.
 
+# This script runs the benchmarks (outside of Krun). setup.sh must have been
```

Comment:
We could call `setup.sh` here? And I'm not sure how useful the comment is, but it doesn't hurt too much either.

Reply:
I'm starting to wonder if these two files would be better as a Makefile. You've git those "if this exists" checks which would be naturally expressed as make dependencies. Similarly, running the experiment could be a dependency of fetching the bits.

Or would this be over-engineering?

Reply:
Over-engineering IMHO.

Reply:
OK.

Let's bear this in mind for the next time we make a build script. I regret `build.sh` in the warmup experiment not being a `Makefile`. It would have been much nicer. Oh well :)